pages/_admin/index.tsx

```diff
@@ -28,7 +31,7 @@ function Story({ id, title, content, name, postal, email, phone, twitter, ...res
           data-type="deleted"
           onClick={(e) => {
             if (window.confirm('Are you sure you want to delete this?')) {
-              updateStory(e)
+              updateStory(e).then()
```

Comment:
What's up with this?

Reply:
Mainly a sanity thing, lint/ts was throwing an error that a promise was being returned and not resolved/handled. I'm open to other solutions though!

Reply:
I agree that this is a bit puzzling. Would it be better to make `updateStory()` not return a promise? I think you could literally just remove the `async` and `await` in this case, and it would work?

Reply:
Trying that out, I can't see why it wouldn't work